addon/components/lazy-popover-on-element.js

```diff
@@ -0,0 +1,129 @@
+import Ember from 'ember';
+import TooltipAndPopoverComponent from 'ember-tooltips/components/tooltip-and-popover';
+
+const { $, run } = Ember;
+
+export default TooltipAndPopoverComponent.extend({
+
+  /* Options */
+  hideDelay: 250,
+
+  /* Properties */
+  classNames: ['ember-popover'],
+  _isMouseInside: false,
+
+  didRender() {
+    // the lazy-render popover component instance needs access to the childView
+    // so that it can call the childView's hide action
+    this._super(...arguments);
+
+    const parentView = this.get('parentView');
+    parentView.set('childView', this);
```

Comment:
I assume there's not a more Emberish way to do this? Maybe by passing an action into the component.

Also, is `childView` too generic here? What si the parent view already has a `childeView` property?